styleguide/source/_patterns/02-organisms/06-topic/06-topic-promotion/topic-promotion.twig

```diff
@@ -0,0 +1,11 @@
+<div class="topic_promotion grid-region_content">
+    {% set tag = promo.cta ? 'div': 'a' %}
+    <{{tag}} class="topic_promotion_container" href="#">
+    {% include 'atoms-h2' with {'content': promo.title} %}
+    {% include 'atoms-rule-horizontal' %}
```

Comment:
In https://github.com/AmericanMedicalAssociation/AMA-style-guide/pull/169 we had consensus to use the rule-horizontal-bottom instead of the rule horizontal atom.